### ISSUE TITLE
CS changed?

### ISSUE THREAD

**Issue by AmnaSN** [git-up/GitUp#1017]
Hi, I wanted to confirm whether the code signature has been changed for the latest version of gitup?

**Comment by core-code** [comment]
all previous releases were signed by ```Pierre-Olivier Latour (88W3E55T4B)``` but 1.4.2 was signed by ```Lucas Derraugh (FP44AY6HHW)```

but it is mentioned in the release notes so it seems ok

**Comment by lucasderraugh** [comment]
TLDR: Yes this is expected.

Long version:
I (Lucas Derraugh) picked the project up from Pierre a long time ago and when I picked up the project he transferred the certificates to me so that I could continue to sign with the same keys. Recently I realized that these certificates were to expire August 3rd so they had to be updated for future versions to be code signed. Pierre no longer holds a developer account and given that I am the maintainer of this project it made sense to use my Apple Developer certificates to sign going forward. We also updated the appcasts to sign the builds so that we could do this key rotation and allow for automatic updates to continue.

@AmnaSN Out of curiosity, did you just notice the release notes or did you notice the actual code signature change somehow?

**Comment by core-code** [comment]
thanks for the explanation!
